### ISSUE TITLE
Banjo to Kazooie no Daibouken 2 (J) - does this

### ISSUE THREAD

**Issue by DerekTurtleRoe** [SimoneN64/Gadolinium#36]
![image](https://user-images.githubusercontent.com/3183676/227806768-5a959300-d55a-479c-a498-ce1972ed871c.png)

Banjo to Kazooie no Daibouken 2 (J) - 5A5172383037D171F121790959962703BE1F373C

This game is a bit weird...it boots, and only shows some models and textures near water. The water animates, and it just stays here and doesn't do anything.

Tested on the latest version of Gadolinium on Windows 10.

**Comment by SimoneN64** [comment]
> ![image](https://user-images.githubusercontent.com/3183676/227806768-5a959300-d55a-479c-a498-ce1972ed871c.png)
> 
> Banjo to Kazooie no Daibouken 2 (J) - 5A5172383037D171F121790959962703BE1F373C
> 
> This game is a bit weird...it boots, and only shows some models and textures near water. The water animates, and it just stays here and doesn't do anything.
> 
> Tested on the latest version of Gadolinium on Windows 10.

I don't understand what this game actually is :smile: is it just Banjo Kazooie 2 from Japan or...?

**Comment by DerekTurtleRoe** [comment]
Yes, it is the Japanese version of Banjo Tooie

**Comment by SimoneN64** [comment]
> Yes, it is the Japanese version of Banjo Tooie

Oh ok, somehow I thought it was a differeny game than Banjo Tooie

**Comment by DerekTurtleRoe** [comment]
> > Yes, it is the Japanese version of Banjo Tooie
> 
> Oh ok, somehow I thought it was a differeny game than Banjo Tooie

Yeah, for reference, I usually use the No-Intro scheme for game titles and such.

So anything with (J) is Japanese, (U) is NTSC, and pretty much everything else is PAL. (A) is Australia, (E) is Europe, (G) is Germany, (F) is France, etc.

But I sometimes use them a bit interchangeably, even though I should always be more specific 😅

**Comment by SimoneN64** [comment]
I can't seem to be able to press any button in any version :(
